etc/p3-config/p3-config.yml

```diff
@@ -197,3 +197,88 @@ p3_flavor_compute_A_magnum:
   disk: 271
   vcpus: 64
   ephemeral: 100
+
+###############################################################################
+# Software images for P-cubed
+
+p3_images:
+  - "{{ p3_image_centos }}"
+  - "{{ p3_image_ubuntu }}"
+  - "{{ p3_image_fedora }}"
+  - "{{ p3_image_openhpc }}"
+  - "{{ p3_image_mlnx_ofed }}"
+
+# Latest CentOS image
+p3_image_centos:
+  name: "CentOS7"
+  elements:
+    - "centos7"
+    - "epel"
+    - "selinux-permissive"
+    #- "alaska-extras"
```

Comment:
It's not in the stackhpc-image-elements repo yet, could do with being incorporated, but it might be doable without a dedicated element.  I'm checking..